mora/service/engagement.py

```diff
@@ -94,10 +94,15 @@ def get_systems(reg):
 )
 @util.restrictargs('at', 'validity', 'start', 'limit')
 def get_engagement(type, id, function):
```

Comment:
One of these days we should really rename this function and move it to somewhere more appropriate.

Reply:
Indeed, yes; the actual design turned out somewhat different from I had envisioned ;)